server/index.mjs

```diff
@@ -1,17 +1,12 @@
-import express from 'express';
-import CONFIG from '../config.js';
-
-
-
 import express from 'express';
 import mysql from 'mysql';
 import path from 'path';
 import cors from 'cors';
 
 const connection = mysql.createConnection({
-  host: 'rightstuff.cvojhsq84htk.us-east-2.rds.amazonaws.com',
-  user: 'rightstuff',
-  password: 'servicepony',
+  host: 'localhost',
+  user: 'root',
+  password: '',
   database: 'rightbar',
 });
 
```

Comment:
Speaking of ambiguity, I need to improve my understanding of code comments and code reviews 😬

Reply:
I actually have a branch where I have moved all of the parameters to an external `config.js` and have started building out my SDC server.

I wanted to spend as little time as possible adjusting the base code I received.  (I would never put in access codes directly into code like this).

Let me know if you think it will be beneficial to me to do this extra work and I'll do it

Reply:
> Speaking of ambiguity, I need to improve my understanding of code comments and code reviews 😬
The green lines are the only lines I changed.  Other code, in white or red, came as seen.